.github/workflows/validate.yml

```diff
@@ -31,3 +31,15 @@ jobs:
             git diff --stat
             exit 1
           fi
+    - name: Scan for accessibility issues
+      task: accessibility-insights.prod.task.accessibility-insights@3
+      condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'))
+      enabled: true
+      inputs:
+        baselineFile: "$(Build.SourcesDirectory)/baselines/vcpkg.github.io.baseline"
+        url: "https://vcpkg.io/en/"
```

Comment:
This should point to the _generated_ files, not the live site since this is meant to catch accessibility bugs in PRs.